### PR TITLE
Clean up of std.uni (part 2)

### PR DESCRIPTION
**Pull request by JackStouffer**
* Removes all total, non-global imports and replaces them with selective imports
* ~~encapsulates a unsafe subtraction from `size_t`~~

### CODE REVIEW

**Comment by wilzbach** [comment]
Wow seems like std.uni get's a lot of love recently!
LGTM!

File: std/uni.d [hunk 1]
```diff
@@ -2835,7 +2837,7 @@ private:
             //  [---+++----++++----++++++]
             //  [                         a  b]
             data.append(a, b);
-            return data.length-1;
+            return data.length > 0 ? data.length - 1 : 0;
```
Comment (aG0aep6G) [inline std/uni.d]: Does this fix a bug? If so, can we get a unittest for it? If not, would `assert(data.length > 0);` be enough?

Comment (JackStouffer) [inline std/uni.d]: The assert is the better solution. I'll add that.

**Comment by JackStouffer** [comment]
After reading the code more, I realized that the check was redundant, so I dropped it.

**Comment by DmitryOlshansky** [comment]
LGTM

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Comment by JackStouffer** [comment]
Thanks!

**Pull request merged**